### PR TITLE
Expand temp space available to Dotmap instances

### LINKED ISSUE #604: Fix Dotmaps temp space issue
Followup to https://github.com/openaddresses/machine/pull/603 and https://github.com/openaddresses/machine/pull/554. Tippcanoe runs out of space:

```
…
2017-04-15 07:55:15,018   DEBUG: Opening /tmp/processed-m96s6eey.zip (us/nd/dickinson)
Read 434.78 million features
                              
Reordering geometry: 0% 
Reordering geometry: 1% 
Reordering geometry: 2% 
…
Reordering geometry: 22% 
Reordering geometry: 23% 
Reordering geometry: 24% 
geom: Write to temporary file failed: No space left on device
```

### PR DESCRIPTION
**Pull request by migurski**
Replaces EC2 instance storage knowledge with dumber `--temp-size` flag to `openaddr-run-ec2-command`.

- [ ] Wait for complete test run of `openaddr-update-dotmap` to complete.

Closes #604.

### CODE REVIEW

**Comment by migurski** [comment]
The first run of this failed when it ran out of time. Trying again with a 24hr cap.

**Comment by migurski** [comment]
The second run is grinding along very slowly after ~300m points, for unknown reasons.

File: chef/crontab/recipes/default.rb [hunk 1]
```diff
@@ -113,7 +113,8 @@
   openaddr-run-ec2-command \
   --role dotmap \
   --hours 16 \
-  --instance-type r3.xlarge \
+  --instance-type r3.large \
```
Comment (NelsonMinar) [inline chef/crontab/recipes/default.rb]: did you mean to pick a smaller instance?

**Comment by NelsonMinar** [comment]
I know nothing about Tippecanoe and EBS. But I did flag the thing up above where you changed the instance size. Other thought: are you sure that `/tmp` is local disk and not some network thing?

File: chef/crontab/recipes/default.rb (hunk 1 above)
Comment (migurski) [inline chef/crontab/recipes/default.rb]: I did! Previously, the larger instance was needed in order to use more temp space, but now temp space is being provided independently of instance storage size.

**Comment by migurski** [comment]
`/tmp` is probably a network thing, due to how AWS does EBS volumes. I peered into `iostat` a bit to see if that was related to the slowness I was seeing, and the disks weren’t being hit very hard at all.

**Comment by migurski** [comment]
Still haven't successfully run this, but I'm going to merge this PR anyway so we can continue generating some logs as I debug this.

**Pull request merged**